network/acceptor_test.go

```diff
@@ -291,12 +304,17 @@ func (suite *AcceptorSuite) TestAcceptorSuccess() {
 	rep := &ReputationManagerMock{}
 	rep.On("Success", address)
 
+	subscriber := make(chan interface{}, 1)
+
 	// act
-	handleAccepting(suite.log, &suite.wg, &suite.cfg, pending, peers, rep, conn)
+	handleAccepting(suite.log, &suite.wg, &suite.cfg, pending, peers, rep, conn, subscriber)
 
 	// assert
 	pending.AssertCalled(suite.T(), "Claim", address)
 	pending.AssertCalled(suite.T(), "Release", address)
 	peers.AssertCalled(suite.T(), "Add", conn, nonce)
 	rep.AssertCalled(suite.T(), "Success", address)
+	event := <-subscriber
+	connected := event.(Connected)
```

Comment:
This will crash if it is the wrong type. Better to use `assert.IsTypeOf`. All asserts return a bool, so inside the condition, you can check for the rest then.